### PR TITLE
Updating 'Creating a Repository'

### PR DESCRIPTION
**Pull request by weaverbel**
Making explicit the need to move to the new directory in order to initialise it.

### CODE REVIEW

**Comment by daisieh** [comment]
Hi Belinda, can you make the corresponding changes in the .md file as well? Once you make the changes there, you should be able to run `make preview` in the directory from the command line and the html file will be automatically generated from the md file.

File: 02-create.html [hunk 1]
```diff
@@ -68,7 +68,7 @@ <h2 id="places-to-create-git-repositories" class="challenge panel panel-success"
 <span class="kw">mkdir</span> beta   <span class="co"># make a sub-directory alpha/beta</span>
 <span class="kw">cd</span> beta      <span class="co"># go into alpha/beta</span>
 <span class="kw">git</span> init     <span class="co"># make the beta sub-directory a Git repository</span></code></pre>
-<p>Why is it a bad idea to do this?</p>
```
Comment (iglpdc) [inline 02-create.html]: The version of this challenge you're changing is outdated. Sorry about that. We messed up at some point and lost the latest version of this challenge. See #96.

**Comment by weaverbel** [comment]
Hi Daisie

I have changed the .md file and created a pull request for it.

https://github.com/swcarpentry/git-novice/pull/100

I actually did this online via the Web site so I have not done step 2 of make preview.

I can do that later if it is needed. Just ran out of time now.

Thanks for your patience with me - I am still learning.


regards
Belinda

Belinda Weaver
eResearch Analyst Team Leader
Queensland Cyber Infrastructure Foundation
Level 5, Axon Building (47)
The University of Queensland
Brisbane AUSTRALIA 4072
e: b.weaver@qcif.edu.au<mailto:b.weaver@uq.edu.au> | p: +617 334 64223 | m: 0408 841 882 | w: www.qcif.edu.au<http://www.qcif.edu.au/> | Twitter @cloudaus<https://twitter.com/cloudaus>
________________________________

**Comment by iglpdc** [comment]
I'm closing this without merging, as there is a new PR (#100) for the markdown. See my comments at #100.

**Pull request closed without merging**